week_6/src/main/scala/org/epfl/ParallelOrNotParallel.scala

```diff
@@ -0,0 +1,64 @@
+package org.epfl.parallel
```

Comment:
I suggest that we use the package `org.epfl.errorhandling.part10` (and a corresponding source directory)